usagov/vote-gov-drupal
# Feature/vote 888/general election date 2024

## Jira ticket
https://cm-jira.usa.gov/browse/VOTE-888

## Description

Add the election date variable corresponding to the deadlines. 
Add the translations for the election date phrase

## Deployment and testing

### Pre-deploy

1. lando retune

### Post-deploy

1. You might have to run lando import-translations

### QA/Test

1. Visit a state page and confirm the date and phrase display above the deadlines.
2. Confirm the translations for the phrase and date are correct

## Checklist for the Developer




## Checklist for the Peer Reviewers
